postgres-types/src/lib.rs

```diff
@@ -1068,3 +1085,9 @@ where
         self
     }
 }
+
+impl<'a, T: ToSql + Send + Sync + 'a> From<T> for Box<dyn ToSql + Send + Sync + 'a> {
+    fn from(to_sql: T) -> Self {
+        Box::new(to_sql)
+    }
+}
```

Comment:
We could also add impl for Sync only, although I'm not sure if/when it would be needed:

```suggestion
}

impl<'a, T: ToSql + Sync + 'a> From<T> for Box<dyn ToSql + Sync + 'a> {
    fn from(to_sql: T) -> Self {
        Box::new(to_sql)
    }
}

```